$NetBSD: patch-Cargo.toml,v 1.2 2024/05/12 19:18:55 wiz Exp $

Don't require higher version of Rust to fix a bug that only affects Windows.

--- Cargo.toml.orig	2024-05-08 06:40:02.000000000 +0000
+++ Cargo.toml
@@ -18,7 +18,7 @@ readme = "README.md"
 repository = "https://github.com/sharkdp/fd"
 version = "10.1.0"
 edition= "2021"
-rust-version = "1.77.2"
+rust-version = "1.76.0"
 
 [badges.appveyor]
 repository = "sharkdp/fd"
